### PR TITLE
Flake in `QuietDownCommandTest.quietDownShouldSuccessWithBlockPlusNonExpiredTimeoutAndRunningExecutor`

### PR DESCRIPTION
**Pull request by jglick**
I noticed a flake in `QuietDownCommandTest.quietDownShouldSuccessWithBlockPlusNonExpiredTimeoutAndRunningExecutor` [here](https://github.com/jenkinsci/jenkins/runs/14797440044) (also showing an apparently unrelated HtmlUnit flake in `ViewTest.privateView` by the way) presumably related to a build left running past the end of the test and interfering with temporary directory cleanup. It turns out that the test utility patched in #8158 was copied (just once, that I can see), so this does an analogous cleanup.

### Testing done

Modified test classes pass locally. Unknown whether the cleanup error can be reproduced.

### Proposed changelog entries

- N/A

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8261"></a>

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
